integration-tests/src/test/groovy/com/okta/sdk/tests/it/spec/UserIT.groovy

```diff
@@ -130,6 +132,10 @@ class UserIT extends ITSupport {
             .setOldPassword(client.instantiate(PasswordCredential).setValue('Abcd1234'))
             .setNewPassword(client.instantiate(PasswordCredential).setValue('1234Abcd')))
         assertThat credentials.provider.type, equalTo(AuthenticationProviderType.OKTA)
+
+        // 3. make the test recording happy, and call a get on the user
+        // TODO: fix har file
+        client.getUser(user.id)
```

Comment:
@vijetmahabaleshwar-okta, i'm not sure why these are needed, is there some additional validation that some of the other SDKs are doing

Reply:
@bdemers - I think after changing password, we're validating that 'last password change time' has increased by calling getUser again.